### ISSUE TITLE
error parsing config

### ISSUE THREAD

**Issue by volodymyrss** [oda-hub/dispatcher-app#445]
this is why the workflow of @andriineronov fails at the moment:

```python
ERROR:cdci_data_analysis.flask_app.dispatcher_query:set_query_exception with Instrument: gaia-andrii, product: Gaia-lightcurve-service failed!
 () during Gaia-lightcurve-service
 [sentry] ERROR: Internal error in sentry_sdk
Traceback (most recent call last):
  File "/pyenv/versions/3.8.5/lib/python3.8/site-packages/cdci_data_analysis/analysis/queries.py", line 370, in test_communication
    test_comm_query_out = instrument.test_communication(config,logger=logger)
  File "/pyenv/versions/3.8.5/lib/python3.8/site-packages/cdci_data_analysis/analysis/instrument.py", line 195, in test_communication
    return self.data_server_query_class(config=config, instrument=self).test_communication(logger=logger)
  File "/pyenv/versions/3.8.5/lib/python3.8/site-packages/dispatcher_plugin_nb2workflow/dataserver_dispatcher.py", line 9, in __init__
    config = DataServerConf.from_conf_dict(instrument.data_server_conf_dict)
  File "/pyenv/versions/3.8.5/lib/python3.8/site-packages/cdci_data_analysis/configurer.py", line 139, in from_conf_dict
    return DataServerConf(required_keys, allowed_keys, **conf_dict)
TypeError: type object argument after ** must be a mapping, not NoneType

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/pyenv/versions/3.8.5/lib/python3.8/site-packages/cdci_data_analysis/analysis/instrument.py", line 385, in run_query
    query_out = query_obj.run_query(self, out_dir, job, run_asynch,
  File "/pyenv/versions/3.8.5/lib/python3.8/site-packages/cdci_data_analysis/analysis/queries.py", line 630, in run_query
    query_out = self.test_communication(instrument,query_type=query_type,logger=logger,config=config,sentry_dsn=sentry_dsn)
  File "/pyenv/versions/3.8.5/lib/python3.8/site-packages/cdci_data_analysis/analysis/queries.py", line 396, in test_communication
    raise InternalError(None)
cdci_data_analysis.analysis.exceptions.InternalError: InternalError()
```

It might be simple, need to check a bit

**Comment by andriineronov** [comment]
Is this because I did something wrong?

I tried to test locally (actually on renku lab) the nb2workflow, when I execute notebook I get an error at the end:

2022-08-29 16:52:08,479 - root - INFO - execution SUCCESSFUL!
[NbConvertApp] Converting notebook /tmp/nb2w-d0pd4448/Gaia-lightcurve-service_output.ipynb to html
/opt/conda/lib/python3.9/site-packages/nbconvert/filters/datatypefilter.py:39: UserWarning: Your element with mimetype(s) dict_keys(['application/scrapbook.scrap.json+json']) is not able to be represented.
  warn("Your element with mimetype(s) {mimetypes}"
[NbConvertApp] Writing 644614 bytes to /home/jovyan/work/gaia-andrii/Gaia-lightcurve-service_output.html
INFO:root:converting... 0
2022-08-29 16:52:10,777 - root - INFO - converting... 0
INFO:nb2workflow.nbadapter:exported html to Gaia-lightcurve-service_output.html
2022-08-29 16:52:10,778 - nb2workflow.nbadapter - INFO - exported html to Gaia-lightcurve-service_output.html
INFO:nb2workflow.nbadapter:validating with ODA dispatcher plugin
2022-08-29 16:52:10,781 - nb2workflow.nbadapter - INFO - validating with ODA dispatcher plugin
WARNING:nb2workflow.nbadapter:unable to import dispatcher_plugin_nb2workflow.queries.NB2WProductQuery: No module named 'dispatcher_plugin_nb2workflow'
2022-08-29 16:52:10,782 - nb2workflow.nbadapter - WARNING - unable to import dispatcher_plugin_nb2workflow.queries.NB2WProductQuery: No module named 'dispatcher_plugin_nb2workflow'

**Comment by andriineronov** [comment]
If I try nb2service, I see that some packages are missing: rfdlib,apscheduler,flask. I tried to pip install them, but then there is an error:

Traceback (most recent call last):
  File "/opt/conda/bin/nb2service", line 5, in <module>
    from nb2workflow.service import main
  File "/opt/conda/lib/python3.9/site-packages/nb2workflow/service.py", line 4, in <module>
    from werkzeug.routing import RequestRedirect, MethodNotAllowed, NotFound
ImportError: cannot import name 'MethodNotAllowed' from 'werkzeug.routing' (/opt/conda/lib/python3.9/site-packages/werkzeug/routing/__init__.py)

I cannot advance here.....

**Comment by volodymyrss** [comment]
> > ImportError: cannot import name 'MethodNotAllowed' from 'werkzeug.routing'
> 
> This should be fixed by [oda-hub/oda_api#153](https://github.com/oda-hub/oda_api/pull/153) But you may need to change the branch, from which the oda_api is installed to `master` (I'm not sure what is your environement on renku currently)
> 
> > dispatcher_plugin_nb2workflow.queries.NB2WProductQuery: No module named 'dispatcher_plugin_nb2workflow'
> 
> Seems that you use this https://github.com/oda-hub/nb2workflow/tree/mmoda-validation branch of nb2workflow. I'm not sure, if it's finished by @volodymyrss Also, I have a question on implementation (in the PR)

That's not related to the title issue, let's move this elsewhere, maybe in the PR.

**Comment by dsavchenko** [comment]
I think this is not actual anymore. I will close.
